docs/blueprints/AMD_coreboot_DRTM_payload.md

```diff
@@ -0,0 +1,98 @@
+# PoC: coreboot with payload started through DRTM (AMD)
```

Comment:
This file is also not referenced anywhere. It should be added to mkdocs or deleted.

Reply:
It is outdated, we're using SLRT now and coreboot code doesn't.

Can you check if the build instructions still work? Testing on hardware isn't necessary at this point. Based on the results we can decide whether to add it or not, and in what form.

Reply:
The build instructions work, but the last commit on the branch it tells to use is almost 2 years ago.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/33/commits/98768b94e7601d5f2fb3f88e4f0ea98b455d1479#diff-caead262477e7246c173717b551fe73aab949ecf77992a80c3be812d664fc0c2R3-R6